backend/src/main/java/org/polsl/backend/entity/ComputerSet.java

```diff
@@ -43,6 +43,40 @@
   public ComputerSet() {
   }
 
+  public String getValidAffiliationName() {
+    for (AffiliationComputerSet affiliationComputerSet : affiliationComputerSetSet) {
+      if (affiliationComputerSet.getValidTo() == null) {
+        return affiliationComputerSet.getAffiliation().getFullName();
+      }
+    }
+    return null;
+  }
+
+  public Set<String> getValidHardwareInventoryNumbers() {
+    Set<String> inventoryNumbers = new HashSet<>();
+
+    computerSetHardwareSet.forEach(computerSetHardware -> {
+      if (computerSetHardware.getValidTo() == null) {
+        inventoryNumbers.add(computerSetHardware.getHardware().getInventoryNumber());
+      }
+    });
+
+    return inventoryNumbers;
+  }
+
+  public Set<String> getValidSoftwareInventoryNumbers() {
+    Set<String> inventoryNumbers = new HashSet<>();
+
+    computerSetSoftwareSet.forEach(computerSetSoftware -> {
+      if (computerSetSoftware.getValidTo() == null) {
+        inventoryNumbers.add(computerSetSoftware.getSoftware().getInventoryNumber());
+      }
+    });
+
+    return inventoryNumbers;
+  }
+
```

Comment:
Spoko, tylko nie jestem pewien, czy to nie powinno być w serwisie, w końcu to logika biznesowa. Jeśli tutaj będzie trochę logiki, a trochę w serwisie, to ciężko to będzie znaleźć. Poszukajmy na necie, czy to dobra praktyka. No chyba, że jesteś pewna, że tak się robi.

Jeśli zdecydujemy się przenieść do do serwisu, to można by też przy okazji przenieść tam inne metody z logiką (te mapujące kolekcje obiektów na kolekcje idków)